### PR TITLE
ci: Retain complete a3p-integration directories and resulting slogfiles

### PR DESCRIPTION
**Pull request by gibson042**
## Description
Extracted from #10165 and dependent upon https://github.com/Agoric/agoric-3-proposals/pull/193 .
This expands `test-docker-build` CI job artifacts from just core eval scripts to their containing a3p-integration directory plus the job's slogfile.

### Security Considerations
n/a

### Scaling Considerations
The final slogfile from #10165 was about 67 MB, which is smaller than the 161 MB deployment-test-results artifact. But regardless, I've limited retention to ~~4 days on success/10 days on failure~~ 10 days, and we could also compress before upload if even that is too much.

### Documentation Considerations
n/a

### Testing Considerations
n/a

### Upgrade Considerations
n/a

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -279,30 +287,57 @@ jobs:
...
-            parent_dir=$(basename "$(dirname "$dir")")
-            # Replace colons with dashes in the parent directory name
-            sanitized_parent_dir=${parent_dir//:/-}
-            # Create the destination directory under /tmp/core_eval_scripts if it doesn't exist
-            destination_dir="/tmp/core_eval_scripts/$sanitized_parent_dir"
-            mkdir -p "$destination_dir"
-            # Copy everything from the $dir to the destination directory
-            cp -r "$dir"/* "$destination_dir/"
-            echo "Copied contents of $dir to $destination_dir"
+          # Copy everything from a3p-integration to /tmp/export/a3p-integration
```
Comment (michaelfig) [inline .github/workflows/integration.yml]: Hmmmm. While it is impressive that you are performing this feat with both hands tied behind your back (with Bash under YAML), would it be easier to call a Node.js script leveraging `execa` instead?

**Review by michaelfig — COMMENTED** [review]
I won't block merging, but approving isn't something I can do while I'm AFK.

File: .github/workflows/integration.yml (hunk 1 above)
Comment (gibson042) [reply .github/workflows/integration.yml]: Meh, no strong feelings either way. I kept it like this as an evolution of what was already there, but am equally happy to change it.

Comment (michaelfig) [reply .github/workflows/integration.yml]: Okay, let's split the difference, and excise the YAML.  Can you factor out the Bash into a file like `agoric-sdk/scripts/$something.sh`, and call it from the workflow?

Comment (gibson042) [reply .github/workflows/integration.yml]: Done.

**Review by michaelfig — APPROVED** [review]
LGTM!  Thanks for extracting `export-a3p.sh`... that really tidies up `.github/workflows/integration.yml`.

**Pull request merged**